helper-cli/src/main/kotlin/utils/Extensions.kt

```diff
@@ -169,7 +170,7 @@ internal fun OrtResult.getLicenseFindingsById(
             packageConfigurationProvider.getPackageConfigurations(id, provenance).flatMap { it.licenseFindingCurations }
         }
 
-    getScanResultsForId(id).filter { it.provenance is KnownProvenance }.map {
+    getScanResultsForId(id).filter { it.provenance is RemoteProvenance }.map {
```

Comment:
Since it is `forEach` and has the `?` operator, I assumed any non-`RepositoryProvenance`s would be skipped by default, which was why `KnownProvenance` was the conditional here, not the stricter `RepositoryProvenance`.